{{cookiecutter.project_slug}}/{{cookiecutter.package_slug}}/extension.py

```diff
@@ -138,4 +138,63 @@ class {{ cookiecutter.project_name|title|replace(" ", "") }}Extension(Extension)
         self.logger.info(f"Obtained custom event with following data: {request}")
         return CustomEventResponse.done()
 
+    {% endif -%}
+
+    {% if cookiecutter.usage_file_process == 'y' -%}
+    {% if cookiecutter.use_asyncio == 'y' %}async {% endif %}def process_usage_file(self, request):  # pragma: no cover
+        self.logger.info(
+            f"Received event for usage file  {request['id']}, type {request['type']} "
+            f"in status {request['status']}",
+        )
+        return ProcessingResponse.done()
+
+    {% endif -%}
+
+    {% if cookiecutter.usage_chunk_file_process == 'y' -%}
+    {% if cookiecutter.use_asyncio == 'y' %}async {% endif %}def process_usage_chunk_file(self, request):  # pragma: no cover
+        self.logger.info(
+            f"Received event for usage chunks file  {request['id']}, type {request['type']} "
+            f"in status {request['status']}",
+        )
+        return ProcessingResponse.done()
+
+    {% endif -%}
+
+    {% if cookiecutter.tier_account_update_request == 'y' -%}
+    {% if cookiecutter.use_asyncio == 'y' %}async {% endif %}def process_tier_account_update_request(self, request):  # pragma: no cover
+        self.logger.info(
+            f"Received event for tier account request  {request['id']}, type {request['type']} "
+            f"in status {request['status']}",
+        )
+        return ProcessingResponse.done()
+
+    {% endif -%}
+
+    {% if cookiecutter.listing_request_process_new == 'y' -%}
+    {% if cookiecutter.use_asyncio == 'y' %}async {% endif %}def process_new_listing_request(self, request):  # pragma: no cover
+        self.logger.info(
+            f"Received event for listing request  {request['id']}, type {request['type']} "
+            f"in status {request['state']}",
+        )
+        return ProcessingResponse.done()
+
+    {% endif -%}
+
+    {% if cookiecutter.listing_request_process_remove == 'y' -%}
+    {% if cookiecutter.use_asyncio == 'y' %}async {% endif %}def process_remove_listing_request(self, request):  # pragma: no cover
+        self.logger.info(
+            f"Received event for listing request  {request['id']}, type {request['type']} "
+            f"in status {request['state']}",
+        )
+        return ProcessingResponse.done()
+
+    {% endif -%}
+
+    {% if cookiecutter.include_schedules_example == 'y' -%}
+    {% if cookiecutter.use_asyncio == 'y' %}async {% endif %}def execute_scheduled_processing(self, schedule):  # pragma: no cover
+        self.logger.info(
+            f"Received event for schedule  {schedule['id']}",
+        )
+        return ProcessingResponse.done()
```

Comment:
In the case of schedulable methods, the response must be ScheduledExecutionResponse.done()

Reply:
You also have to import ScheduledExecutionResponse

Reply:
thanks, fixed this

Reply:
modified also docker-compose.yml